docker-compose.yaml

```diff
@@ -0,0 +1,27 @@
+version: "3"
+services:
+  llama-cpp:
+    build:
+      context: .
+    hostname: "${LLAMA_CPP_HOST}"
+    command: "python3 -m llama_cpp.server --model ${MODEL_FILEPATH} --host 0.0.0.0 --port ${LLAMA_CPP_SERVER_PORT}"
+    stdin_open: true
+    tty: true
+    env_file: .env
+    ports:
+      - "${LLAMA_CPP_SERVER_PORT-8090}:${LLAMA_CPP_SERVER_PORT-8090}"
```

Comment:
There should be `:` here. And for other defaults as well.

Reply:
I'd appreciate if you explained why the ":" should be there. Both are valid syntax.

> ${VAR:-default} -> value of VAR if set and non-empty, otherwise default
> ${VAR-default} -> value of VAR if set, otherwise default

I don't have strong preference either way because no one will touch this anyway.

Reply:
Well, an empty string will probably break stuff if supplied as host parameter, or port number. It's true that that nobody will override these.